src/ExpressReceiver.ts

```diff
@@ -30,17 +33,28 @@ export default class ExpressReceiver extends EventEmitter implements Receiver {
 
   private server: Server;
 
+  private axios: AxiosInstance;
+
   constructor({
     signingSecret = '',
     logger = new ConsoleLogger(),
-    endpoints = { events: '/slack/events' }
+    endpoints = { events: '/slack/events' },
+    agent = undefined,
+    clientTls = undefined,
   }: ExpressReceiverOptions) {
     super();
 
     this.app = express();
     this.app.use(this.errorHandler.bind(this));
     // TODO: what about starting an https server instead of http? what about other options to create the server?
     this.server = createServer(this.app);
+    this.axios = axios.create(Object.assign(
```

Comment:
Ah, I overlooked the necessity to support the `respond` function as well. Great catch 🙇